docs/conf.py

```diff
@@ -0,0 +1,155 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+import nemos
+import sys, os
+from pathlib import Path
+
+sys.path.insert(0, str(Path('..', 'src').resolve()))
+sys.path.insert(0, os.path.abspath('sphinxext'))
+
+
+project = 'nemos'
+copyright = '2024'
+author = 'E Balzani'
```

Comment:
Sure, I don't mind that we should have a discussion about this type of things. I think @gviejo  is more of the opinion that one would benefit more if credited individually on something like the landing page. I see the point as well, because people will read your name and associate it to the project, while "nemos authors" is more fare but needs an extra step to figure out who these people are.